web_application/Backend/FlaskApp/app.py

```diff
@@ -431,34 +430,55 @@ def send_invitation():
         data = request.get_json()
         receiver_id = data.get('receiver_id')
         phenotype = data.get('phenotype')
+        collaboration_uuid = data.get('collaboration_uuid')
```

Comment:
```suggestion
        collaboration_id = data.get('collaboration_id')
```

Reply:
Let's use xxx_id everywhere. UUID would be confusing if we use `receiver_id` or something at the same time.

Reply:
Note.